addon/components/-private/base-pop-menu.js

```diff
@@ -205,9 +225,13 @@ export default class BasePopMenuComponent extends Component {
     if (this.get('triggerEvent') === 'hover') {
       this._popperElement.removeEventListener('mouseenter', this._openPopoverHandler);
       this._popperElement.removeEventListener('mouseleave', this._closePopoverHandler);
+      this._popperElement.removeEventListener('focus', this._openPopoverHandler);
+      this._popperElement.removeEventListener('blur', this._closePopoverBlurHandler);
     }
 
+    this._popperElement.removeEventListener('keydown', this._popperKeyHandler);
```

Comment:
FYI for reviewers, this is the block that's causing the test fail in ember 1.13. It has something to do with `this._popperElement = null;` executing before the rest of it, so it throws the null error.

Reply:
I think it might be because of something else. Try test `dropdown box closes when element outside of dropdown is clicked`. That test fails and never gets here.

Reply:
Failure appears to be because of changes in `@argument`, diving in this morning to try to figure it out